zipline/gens/brokers/rh/robinhood.py

```diff
@@ -0,0 +1,1147 @@
+                (str): stock symbol
+        """
+
+        return self.get_quote_list(stock,'symbol')
+
+
+    def last_updated_at(self, stock=''):
+        """Get last update datetime
+
+            Note:
+                queries `quote` endpoint, dict wrapper
+
+            Args:
+                stock (str): stock ticker
+
+            Returns:
+                (str): last update datetime
+        """
+
+        return self.get_quote_list(stock, 'last_updated_at')
+
+    
+    def last_updated_at_datetime(self, stock=''):
+        """Get last updated datetime
+
+            Note:
+                queries `quote` endpoint, dict wrapper
+                `self.last_updated_at` returns time as `str` in format: 'YYYY-MM-ddTHH:mm:ss:000Z'
+
+            Args:
+                stock (str): stock ticker
+
+            Returns:
+                (datetime): last update datetime
+
+        """
+
+        #Will be in format: 'YYYY-MM-ddTHH:mm:ss:000Z'
+        datetime_string = self.last_updated_at(stock)
+        result = dateutil.parser.parse(datetime_string)
+
+        return result
+
+
+    def get_account(self):
+        """Fetch account information
+
+            Returns:
+                (:obj:`dict`): `accounts` endpoint payload
+        """
+
+        res = self.session.get(self.endpoints['accounts'])
+        res.raise_for_status()  #auth required
+        res = res.json()
+
+        return res['results'][0]
+
+
+    def get_url(self, url):
+        """
+            Flat wrapper for fetching URL directly
+        """
+
+        return self.session.get(url).json()
+
+
+    ###########################################################################
+    #                           GET FUNDAMENTALS
+    ###########################################################################
+
+    def get_fundamentals(self, stock=''):
+        """Find stock fundamentals data
+
+            Args:
+                (str): stock ticker
+
+            Returns:
+                (:obj:`dict`): contents of `fundamentals` endpoint
+        """
+
+        #Prompt for stock if not entered
+        if not stock:   #pragma: no cover
+            stock = input("Symbol: ")
+
+        url = str(self.endpoints['fundamentals']) + str(stock.upper()) + "/"
+
+        #Check for validity of symbol
+        try:
+            req = requests.get(url)
+            req.raise_for_status()
+            data = req.json()
+        except requests.exceptions.HTTPError:
+            raise RH_exception.InvalidTickerSymbol()
+
+
+        return data
+
+
+    def fundamentals(self, stock=''):
+        """Wrapper for get_fundamentlals function """
+
+        return self.get_fundamentals(stock)
+
+
+    ###########################################################################
+    #                           PORTFOLIOS DATA
+    ###########################################################################
+
+    def portfolios(self):
+        """Returns the user's portfolio data """
```

Comment:
In general: I'd rather not put doc-strings unless it is necessary. This comment does not help to understand what's returned. Please either remove it or make it meaningful.